src/bundler.ts

```diff
@@ -393,7 +393,7 @@ export class Bundler {
       ast: ASTNode,
       bundle: AssignedBundle,
       bundleManifest: BundleManifest) {
-    const visitedUrls = new Set<UrlString>();
+    const visitedUrls = new Set<UrlString>(bundle.bundle.stripImports);
```

Comment:
Actually I went and renamed the variable to `stripImports` as well as in the inlining function along with adjustments to comments.  "visited" is too abstract anyways.